Detections/VMwareCarbonBlack/CriticalThreatDetected.yaml

```diff
@@ -17,13 +17,13 @@ relevantTechniques:
   - T1210
 query: |
 
-  let threshold = 8;
-  CarbonBlackNotifications_CL
-  | where threatHunterInfo_score_d >= threshold
-  | extend eventTime = datetime(1970-01-01) + tolong(threatHunterInfo_time_d/1000) * 1sec
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by eventTime, Threat_Name = threatHunterInfo_reportName_s, Device_Name = deviceInfo_deviceName_s,  Internal_IP = deviceInfo_internalIpAddress_s, External_IP = deviceInfo_externalIpAddress_s, Threat_Score = threatHunterInfo_score_d
-  | project-away count_
-  | extend timestamp = StartTime, HostCustomEntity = Device_Name, IPCustomEntity = Internal_IP
+let threshold = 8;
```

Comment:
You need to add 2 spaces to the front of each line under the query: | block otherwise the YAML structure is broken.  You can use any online YAML validator to make sure the YAML structure is correct before submitting your PR.